bin/citgm-all.js

```diff
@@ -47,6 +56,17 @@ if (!app.su) {
   log.warn('root', 'Running as root! Use caution!');
 }
 
+if (app.withTags){
+  app.withTags = app.withTags.split(',');
```

Comment:
this won't be necessary if we us the [array key](https://www.npmjs.com/package/yargs#arraykey)